core/domain/user_services.py

```diff
@@ -170,6 +171,16 @@ def validate(self):
         if self.role not in role_services.PARENT_ROLES:
             raise utils.ValidationError('Role %s does not exist.' % self.role)
 
+        if not isinstance(self.creator_dashboard_display, basestring):
+            raise utils.ValidationError(
+                'Expected dashboard display preference to be a string,' \
```

Comment:
Don't use backslashes at end of lines. You can just remove them, I think -- if you do

    'string A'
    'string B'

in parens, then Python automatically concatenates them. Remember to leave a space after the comma at the end of the first line.